File: src/main/java/in/dragonbra/javasteam/util/log/LogListener.java
package in.dragonbra.javasteam.util.log;

/**
 * @author lngtr
 * @since 2018-03-02
 */
public interface LogListener {
    void onLog(Class<?> clazz, String message, Throwable throwable);

    void onError(Class<?> clazz, String message, Throwable throwable);
}

File: src/main/java/in/dragonbra/javasteam/util/log/LogListener.kt
package `in`.dragonbra.javasteam.util.log

/**
 * @author lngtr
 * @since 2018-03-02
 */
interface LogListener {
    fun onLog(clazz: Class<*>, message: String?, throwable: Throwable?)
    fun onError(clazz: Class<*>, message: String?, throwable: Throwable?)
}
